CloudAppSecurityDocs/app-governance-predefined-policies.md

```diff
@@ -21,10 +21,7 @@ App governance contains a set of out of the box policies to detect anomalous app
 
 - To view the description of the policy, select the policy to see the policy summary and description in the detailed policy window.
 - To change the status of a policy (deactivate / activate), select the policy and select **Deactivate** in the detailed policy window.
-- You can view the total number of active predefined policies in the policy snapshot.
-
-    ![View number of policies.](media/app-governance/number-policies.png)
-
+- By default, predefined policies will trigger alerts when the conditions are met. You can choose to automatically Disable app when the policy triggers. Use caution when applying actions because a policy may affect users and legitimate app use. To disable app, mark Disable app box under Policy action in the summary section and click save.
```

Comment:
```suggestion
- By default, predefined policies will trigger alerts when the conditions are met. You can choose to automatically disable the app when the policy triggers. Use caution when applying these actions because a policy may affect users and legitimate app use. To disable the app, mark the **Disable app** box under **Policy action** in the summary section and select **Save**.
```